src/inject.js

```diff
@@ -99,10 +100,18 @@ const stringHandlers = {
             // Since keyframes need 3 layers of nesting, we use `generateCSS` to
             // build the inner layers and wrap it in `@keyframes` ourselves.
             let finalVal = `@keyframes ${name}{`;
-            Object.keys(val).forEach(key => {
-                finalVal += generateCSS(
-                    key, [val[key]], selectorHandlers, stringHandlers, false);
-            });
+
+            if (val instanceof OrderedElements) {
```

Comment:
I'm not sure if we can change things so that this fork doesn't need to happen. If we can't it might be worth adding a utility function that takes either a plain object or OrderedElements instance and iterates over the keys/values for convenience.